runtime.go

```diff
@@ -4,8 +4,10 @@ import (
 	"code.google.com/p/go.net/websocket"
 	"github.com/nu7hatch/gouuid"
 	"log"
-	"net"
+	//"net"
```

Comment:
One generic note on the changes to `runtime.go`: please consider that the entire `flow` package is supposed to be used by many different applications, so hardcoding anything application-specific like names, test output and hardcoded component mappings is ineligible. Application-specific details are either passed explicitly by the client application that imports the package, or deduced by the package by the means of reflection (if possible/appropriate).